### PR TITLE
Reduce height of Brand and Script Link components

### LINKED ISSUE #2555: Reduce height of Brand and Script Link components
**Is your feature request related to a problem? Please describe.**
The new brand designs have reduced the height of the brand component on breakpoints below 400px and the brand and script link components above 600px.

**Describe the solution you'd like**
Reduce the height of the components to match the designs.

**Testing notes**
- Manual testing in psammead storybook
- Ensure grid fallback working as expected in IE11

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

image

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #2555 

**Overall change:**
Changed the height of Script Link component for breakpoints above 600px and the height of Brand Component for all breakpoints.

**Code changes:**
- Changed styling for Script Link and Brand Component
- Updated snapshots

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by tochwill — DISMISSED** [review]

**Comment by DenisHdz** [comment]
The `Brand` doesn't look to be vertically centered inside the Banner. Is that expected?

brand

brand2

**Comment by OlgaLyubin** [comment]
@DenisHdz 
Originally the Brand was implemented in such a way that it is centred, but top and bottom paddings work in such a way that bottom padding is 4px smaller than the top padding, however it's compensated with the height of the piece of red banner underneath. I guess this is because if you hover over the Brand, that little piece of red banner becomes white to signify a link. So in terms of pixels, the Brand is centred within the Banner. Here is a screenshot of Storybook from latest : 

Screenshot 2019-11-07 at 12 19 16

I just followed the same logic in my PR.

**Comment by DenisHdz** [comment]
I think the min-height in [this](https://github.com/bbc/psammead/pull/2569/files#diff-4f91a28dd0d1d26145e763893261c522R58) media query needs to be updated because the separation at the top and bottom of the banner under 240px is not proportional:
min-height-with-script-link

On the other hand, we should apply that media query just when the `ScriptLink` is passed, otherwise, we will be incrementing the height when is not needed, resulting in this:

min-height-without-script-link

File: packages/components/psammead-brand/src/index.jsx [hunk 1]
```diff
@@ -54,7 +57,12 @@ const Banner = styled.div`
   }
 
   @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
-    min-height: ${({ svgHeight }) => svgHeight / 16 + 5}rem;
+    ${({ scriptLink, svgHeight }) =>
+      scriptLink !== null &&
```
Comment (DenisHdz) [inline packages/components/psammead-brand/src/index.jsx]: Thanks for the changes. One last thing:

```suggestion
      scriptLink &&
```

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by Bopchy — APPROVED** [review]

**Review by ryanmccombe — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Looks good. We should also get this UX reviewed prior to merging.

**Review by tochwill — APPROVED** [review]

**Comment by OlgaLyubin** [comment]
This PR has already been reviewed and approved by UX

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**